.github/workflows/deploy.yaml

```
@@ -3,9 +3,8 @@ name: Deploy production
 on:
   workflow_dispatch: ~
   push:
-    branches: [main]
-  schedule:
-    - cron: '0 8 * * 1,2,3,4,5' # At 08:30 (UTC, => 09:30 / 10:30 in Europe/Paris depending on the DST), each day of the week.
+    tags:
+      - v2.**
```

```suggestion
      - v2.*
```

devrait suffire: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags

non ?

---

Complètement, je ne sais pas d'où j'ai sorti ça...